tests/integration/model/test_models.py

```diff
@@ -18,11 +18,90 @@
 from anomalib.utils.types import TaskType
 
 
-def models() -> list[str]:
+def models() -> set[str]:
     """Return all available models except ai_vad."""
     return get_available_models()
 
 
+def _get_objects(
```

Comment:
Any particular reason this is moved out of the class?

Reply:
To use as a utility in export tests

Reply:
In this particular case, if `_get_objects` needs to be used by other tests it would be better to move it to `conftest.py`
But I would suggest waiting till we merge https://github.com/openvinotoolkit/anomalib/pull/1514 as there is some overlapping work here.